src/views/Balance.vue

```diff
@@ -1,30 +1,225 @@
 <template>
     <h2 class="text-gray-400 text-3xl font-medium">Balance</h2>
     <p class="text-gray-400">Account: {{ allAccounts ? allAccounts[0] : 'no account selected' }}</p>
-    <!-- <p class="text-gray-400">Balance: {{ balance ? balance.toString() : '0' }}</p> -->
+    <p class="text-gray-400">Balance: {{ balance ? balance.toString(10) : 'cannot read' }}</p>
     <p class="text-gray-400">Counter: {{ testCounter ? testCounter.toString() : '0' }}</p>
 
     <p class="text-gray-400">{{ api ? api.isConnected : 'no api' }}</p>
+    <div class="flex flex-col flex-1 overflow-y-auto lg:overflow-hidden">
+        <!-- <%- include('./components/common/_header-mobile'); -%> -->
+
+        <main class="flex-1 relative z-0 lg:overflow-y-auto overflow-x-hidden focus:outline-none">
+            <div class="px-4 sm:px-8 py-10 lg:py-16">
+                <div
+                    class="sm:flex items-end border-b border-gray-300 dark:border-darkGray-600 mb-8 -mx-4 sm:-mx-8 px-4 sm:px-8"
+                >
+                    <h1 class="text-3xl font-extrabold text-blue-900 dark:text-white mb-6 sm:mb-8">
+                        dApps
+                    </h1>
+                    <Tab :labels="['dApps Staking', 'Create dApps']" />
```

Comment:
let's make the naming scheme consistent by using the kebab-case for template elements.